packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -784,22 +879,6 @@ const Routes = ({
     });
   };
 
-  const routesToSave = routes.map((route) => {
-    if (isRouter(route)) {
-      return route;
-    }
-    return {
-      id: route.id,
-      attributeRoutingConfig: route.attributeRoutingConfig,
-      action: route.action,
-      isFallback: route.isFallback,
-      queryValue: route.queryValue,
-      attributesQueryValue: route.attributesQueryValue,
-    };
-  });
-
-  hookForm.setValue("routes", routesToSave);
```

Comment:
Needed to watch for `routes` in SingleForm which made me aware that we are doing `setValue` on render causing infinite rendering.

Now, we do it on user action of making changes in the route-builder